src/cascadia/TerminalSettingsModel/KeyChordSerialization.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "KeyChordSerialization.g.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Model::implementation
+{
+    struct KeyChordSerialization
+    {
+        KeyChordSerialization() = default;
+
+        static winrt::Microsoft::Terminal::TerminalControl::KeyChord FromString(const winrt::hstring& str);
+        static winrt::hstring ToString(const winrt::Microsoft::Terminal::TerminalControl::KeyChord& chord);
+    };
+}
+
+namespace winrt::Microsoft::Terminal::Settings::Model::factory_implementation
+{
+    struct KeyChordSerialization : KeyChordSerializationT<KeyChordSerialization, implementation::KeyChordSerialization>

```

Comment:
do you need a factory for this one at all? it doesn't have any methods

Reply:
if so, switch to `BASIC_FACTORY`

Reply:
Yeah :/ this was a surprise for me too.